### PR TITLE
Adding CVE-2020-8910

### PR DESCRIPTION
**Pull request by u269c**
Committing CVE-2020-8910 on Closure-Library.

https://github.com/google/closure-library/releases/tag/v20200315

### CODE REVIEW

File: 2020/8xxx/CVE-2020-8910.json [hunk 1]
```diff
@@ -1,18 +1,110 @@
 {
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8910.json]: 'version_value' is a required property

File: 2020/8xxx/CVE-2020-8910.json [hunk 2]
```diff
@@ -1,18 +1,111 @@
...
```
Comment (cve-team) [inline 2020/8xxx/CVE-2020-8910.json]: MISC or CONFIRM, but name != url for reference.

**Comment by shicks** [comment]
LGTM, thanks for pushing this.

**Comment by cve-team** [comment]
Thanks for your submission. This ID has been populated and will be published on cve.mitre.org within 1-2 hours.

**Pull request merged**